fastlane/metadata/steam/czech/achievements.vdf

```diff
@@ -0,0 +1,109 @@
+		"NEW_ACHIEVEMENT_20_17_NAME"	"Distributor v zácviku"
+		"NEW_ACHIEVEMENT_20_17_DESC"	"Vynalezni Směrovač."
+		"NEW_ACHIEVEMENT_20_18_NAME"	"Stvořitel"
+		"NEW_ACHIEVEMENT_20_18_DESC"	"Umísti 10.000 bloků."
+		"NEW_ACHIEVEMENT_20_19_NAME"	"Viktor čistič"
+		"NEW_ACHIEVEMENT_20_19_DESC"	"Znič 1.000 nepřítelských bloků."
+		"NEW_ACHIEVEMENT_20_20_NAME"	"Velkolepá chyba"
+		"NEW_ACHIEVEMENT_20_20_DESC"	"Způsob přetížení Thoriového reaktoru, následované explozí."
+		"NEW_ACHIEVEMENT_20_21_NAME"	"Kartograf"
+		"NEW_ACHIEVEMENT_20_21_DESC"	"Vytvoř 10 nových map."
+		"NEW_ACHIEVEMENT_20_22_NAME"	"Sosač"
+		"NEW_ACHIEVEMENT_20_22_DESC"	"Stáhni mapu z dílny."
+		"NEW_ACHIEVEMENT_20_23_NAME"	"Tvůrce"
+		"NEW_ACHIEVEMENT_20_23_DESC"	"Publikuj mapu do dílny."
+		"NEW_ACHIEVEMENT_20_24_NAME"	"Řezník"
+		"NEW_ACHIEVEMENT_20_24_DESC"	"Poraž Záporáka."
+		"NEW_ACHIEVEMENT_20_25_NAME"	"Průzkumník"
+		"NEW_ACHIEVEMENT_20_25_DESC"	"Odemkni všechny zóny v kampani."
+		"NEW_ACHIEVEMENT_20_26_NAME"	"Perfektionalista"
+		"NEW_ACHIEVEMENT_20_26_DESC"	"Dosáhni požadavku pro vlastní výbavu ve všech zónách v kampani."
+		"NEW_ACHIEVEMENT_20_29_NAME"	"Materiály II"
+		"NEW_ACHIEVEMENT_20_29_DESC"	"Odemkni Thorium."
+		"NEW_ACHIEVEMENT_20_31_NAME"	"Materiály I"
+		"NEW_ACHIEVEMENT_20_31_DESC"	"Odemkni Titan."
+		"NEW_ACHIEVEMENT_21_0_NAME"	"Kamikaze"
+		"NEW_ACHIEVEMENT_21_0_DESC"	"Naplň svého mecha výbušným materiálem a zemři v explozi ."
+		"NEW_ACHIEVEMENT_21_1_NAME"	"A tak to začíná"
+		"NEW_ACHIEVEMENT_21_1_DESC"	"Vytvoř továrnu na mecha Dýka."
+		"NEW_ACHIEVEMENT_21_2_NAME"	"Přímý útok"
+		"NEW_ACHIEVEMENT_21_2_DESC"	"Vyšli příkaz k útoku z řídícího střediska."
+		"NEW_ACHIEVEMENT_21_3_NAME"	"Roj"
+		"NEW_ACHIEVEMENT_21_3_DESC"	"Měj v jeden ukamžil 100 aktivních jednotek."
+		"NEW_ACHIEVEMENT_21_4_NAME"	"Stádo"
+		"NEW_ACHIEVEMENT_21_4_DESC"	"Měj v jeden okamžik 10 aktivních dronů Fantóm ."
+		"NEW_ACHIEVEMENT_21_5_NAME"	"Proměnlivá armáda"
+		"NEW_ACHIEVEMENT_21_5_DESC"	"Měj v jeden okamžik 50 aktivních jednotek Slídič."
+		"NEW_ACHIEVEMENT_21_6_NAME"	"Legie"
+		"NEW_ACHIEVEMENT_21_6_DESC"	"Postav celkem 1.000 jednotek."
+		"NEW_ACHIEVEMENT_21_7_NAME"	"Super"
+		"NEW_ACHIEVEMENT_21_7_DESC"	"Dosáhni hodnocení S v jakékoliv zóně."
+		"NEW_ACHIEVEMENT_21_8_NAME"	"Super Super"
+		"NEW_ACHIEVEMENT_21_8_DESC"	"Dosáhni hodnocení SS v jakékoliv zóně."
+		"NEW_ACHIEVEMENT_21_9_NAME"	"Měl jsi líp poslouchat"
+		"NEW_ACHIEVEMENT_21_9_DESC"	"Zemři v zóně pro vylíhnutí nepřítelských jednotek."
+		"NEW_ACHIEVEMENT_21_10_NAME"	"Prostě zmáčkni Shift"
+		"NEW_ACHIEVEMENT_21_10_DESC"	"Nějak se utop."
+		"NEW_ACHIEVEMENT_21_11_NAME"	"Sběratel"
+		"NEW_ACHIEVEMENT_21_11_DESC"	"Naplň jádro na maximálná kapacitu každým typem materiálu."
+		"NEW_ACHIEVEMENT_21_12_NAME"	"Dav"
+		"NEW_ACHIEVEMENT_21_12_DESC"	"Hostuj server s aspoŇ 10 hráči."
+		"NEW_ACHIEVEMENT_21_13_NAME"	"Nezranitelný"
+		"NEW_ACHIEVEMENT_21_13_DESC"	"Postav Tavičku a Ducha."
```

Comment:
Nejsem si úplně jistý, jestli Tavičku a Ducha překládat. Mám pocit, že Meltdown & Spectre jsou reference na security problémy v processorech ze začátku 2018 a pokud se to přeloží, nikdy to nikomu nesepne a reference se ztratí v překladu. :-)

Reply:
vidíš, to mě nenapadlo, a to jsme ajťák :)) ok, nechám to tam jako meltdown a spectre, i když si myslím, že jen málo lidí to docení :))

Reply:
btw políci to mají jako Duch a Rozpad :)